modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-host-practices.adoc
+
+[id='create-a-kubeletconfig-crd-to-edit-kubelet-parameters-{context}']
+= Create a KubeletConfig CRD to edit kubelet parameters
+
+The kubelet configuration is currently serialized as an ignition configuration,
+so it can be directly edited. However, there is also a new
+kubelet-config-controller added to the Machine Config Controller (MCC). This
+allows you to create a KubeletConfig CustomResourceDefinition (CRD) to edit the
+kubelet parameters.
+
+.Procedure
+
+. Run:
++
+----
+$ oc get machineconfig
+----
++
+This provides a list of the available machine configuration objects you can
+select. By default, the two kubelet-related configs are `01-master-kubelet` and
+`01-worker-kubelet`.
+
+. To set the max Pods per node on the worker nodes, make a file. For example,
```

Comment:
maybe s/make a file. For/create a YAML file that contains the kublet configuration, for

Reply:
That's fine with me